kuma/users/signup.py

```diff
@@ -43,6 +46,9 @@ def __init__(self, *args, **kwargs):
             'min_length': USERNAME_SHORT,
             'max_length': USERNAME_LONG,
         }
+        # Add recaptcha if configured in settings.
+        if getattr(settings, 'RECAPTCHA_PRIVATE_KEY', None):
```

Comment:
It would be better to make this a constance config to be able to deploy this without webops help similarly how we did with the Akismet API key